plugins/InspectionGadgets/InspectionGadgetsAnalysis/src/com/siyeh/ig/javadoc/UnnecessaryInheritDocInspection.java

```diff
@@ -168,7 +169,7 @@ else if (owner instanceof PsiMethod) {
           return;
         }
       }
-      registerError(tag, WarningType.EMPTY);
+      registerError(tag, ProblemHighlightType.LIKE_UNUSED_SYMBOL, WarningType.EMPTY);
```

Comment:
it would be not visible in javadoc, I think

Reply:
As you like. I rollback.